cmd/run.go

```diff
@@ -332,11 +347,20 @@ func RunServer(ctx context.Context, config *Config) error {
 
 	logger := buildLogger(config.Log.Format)
 	tracer := telemetry.NewNoopTracer()
-	meter := telemetry.NewNoopMeter()
 	tokenEncoder := encoder.NewBase64Encoder()
 
-	var datastore storage.OpenFGADatastore
 	var err error
+	var meter metric.Meter
```

Comment:
meter := metrics.NewNoopMeter()

then set it if `config.Metrics.Enabled`.

No need for the `else {..}` below.

Reply:
✅